modules/visual_script/editor/visual_script_editor.cpp

```diff
@@ -38,11 +38,11 @@
 #include "editor/editor_node.h"
 #include "editor/editor_resource_preview.h"
 #include "editor/editor_scale.h"
+#include "modules/visual_script/visual_script_expression.h"
```

Comment:
I think it would be better to use relative paths so that the module can be renamed and see till work fine.